platform/integrations/about_integrations.mdx

```diff
@@ -1,18 +1,15 @@
----
-id: integration
-title: Integration
-sidebar_label: Integration
-slug: /integration
----
-
-Easiest way to integrate Tolgee with your application is to go to integration guide, directly in project menu.
-
-![img](/img/docs/platform/project_menu_integrate.png)
-
-Follow the steps for your desired library or framework:
-
-![img](/img/docs/platform/integration_libraries.png)
-
-This way you can easily integrate new project including creating API keys for Tolgee library.
-
-
+---
+id: about_integrations
+title: About Integrations
+sidebar_label: About Integrations
+---
+
+Easiest way to integrate Tolgee with your application is to go to integration guide, directly in project menu.
+
+![img](/img/docs/platform/project_menu_integrate.png)

```

Comment:
This is a screenshot from old design, I guess it can be removed, becuse from the following screenshot it's pretty clear.